fbw-common/src/systems/shared/src/FbwAircraftSentryClient.ts

```diff
@@ -161,8 +161,8 @@ export class FbwAircraftSentryClient {
     const popup = new PopUpDialog();
 
     return new Promise<boolean>((resolve) => {
-      popup.showPopUp(
-        'A32NX - ERROR REPORTING',
+      popup.showPopUp( /
```

Comment:
```suggestion
      popup.showPopUp(
```

The `/` character is causing the build to fail. Remove to correct the syntax please.